### ISSUE TITLE
Bug: `set_parameter` will ignore everything but the first parameter

### ISSUE THREAD

**Issue by mojomex** [tier4/nebula#108]
Some callbacks for parameter changes (registered via `rclcpp::Node::add_on_set_parameters_callback`) use short-circuit `||` operators to chain multiple `get_param` calls, e.g.:

```c++
get_param(p, "sensor_model", sensor_model_str) ||
get_param(p, "return_mode", return_mode_str) || 
get_param(p, "host_ip", new_param.host_ip)
```

As soon as one of those parameters is found, `get_param` returns `true`, thus stopping execution early (short-circuit operator).

The correct operator to use would be the `|` operator, which does not short-circuit and thus all `get_param` statements are executed:

```c++
get_param(p, "sensor_model", sensor_model_str) |
get_param(p, "return_mode", return_mode_str) |
get_param(p, "host_ip", new_param.host_ip)
```

Functions exhibiting this behavior:
* [HesaiHwInterfaceRosWrapper::paramCallback](https://github.com/tier4/nebula/blob/73a1e1b9daacab4dc16774f63420e0e9be630bc6/nebula_ros/src/hesai/hesai_hw_interface_ros_wrapper.cpp#L373-L398)
* [VelodyneHwInterfaceRosWrapper::paramCallback](https://github.com/tier4/nebula/blob/73a1e1b9daacab4dc16774f63420e0e9be630bc6/nebula_ros/src/velodyne/velodyne_hw_interface_ros_wrapper.cpp#L248-L271)

**Comment by mojomex** [comment]
Will be fixed once PRs
* #127 
* #148 
* #147 
* #151 

are merged into `main`.

**Comment by mojomex** [comment]
Fixed by #127, #148, #147, #151.
